infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -122,11 +123,14 @@ type RegistryPersistence struct {
 }
 
 // RegistryFilePersistence configures the file-based persistence for the registry service
-// +kubebuilder:validation:XValidation:rule="(!has(self.pvc) && has(self.path)) ? self.path.startsWith('/') : true",message="Ephemeral stores must have absolute paths."
+// +kubebuilder:validation:XValidation:rule="(!has(self.pvc) && has(self.path)) ? (self.path.startsWith('/') || self.path.startsWith('s3://') || self.path.startsWith('gs://')) : true",message="Ephemeral stores must use absolute paths or be S3 ('s3://') or GS ('gs://') object store URIs."
```

Comment:
Do we need to use ephemeral when we are referring the s3 or gs bucket?

Reply:
you are right, trying a more generic `Registry files`, thanks